File: service/src/main/java/by/rekuts/travelagency/service/ReviewService.java
package by.rekuts.travelagency.service;

import by.rekuts.travelagency.domain.Review;

/**
 * Interface ReviewService concerns to Service layer.
 * Realisations of this interface has access to DAO layer
 * in the face of ReviewRepository interface and could call its methods.
 */
public interface ReviewService extends CrudService<Review> {
}

File: service/src/main/scala/by/rekuts/travelagency/service/ReviewService.scala
package by.rekuts.travelagency.service

import by.rekuts.travelagency.domain.Review

/**
  * Trait ReviewService concerns to Service layer.
  * Realisations of this trait has access to DAO layer
  * in the face of ReviewRepository trait and could call its methods.
  */
trait ReviewService extends CrudService[Review]
